### PR TITLE
Adjust InputLIRC to match new InputComponent.

### PR DESCRIPTION
**Pull request by LongChair**
This fixes the autorepeat issue with IR & autorepeat

### CODE REVIEW

**Comment by plexappbuilder** [comment]
:balloon: - Good work!

File: src/input/InputComponent.h [hunk 1]
```diff
@@ -84,6 +84,7 @@ class InputComponent : public ComponentBase
   const char* componentName() override { return "input"; }
   bool componentExport() override { return true; }
   bool componentInitialize() override;
+  bool supportsKeyUpDown() { return true; }
```
Comment (tru) [inline src/input/InputComponent.h]: you should mark this as `override` it's the C++11 version of virtual.

File: src/input/InputLIRC.h [hunk 2]
```diff
@@ -22,6 +22,7 @@ class InputLIRC : public InputBase
 
   virtual const char* inputName() { return "LIRC"; }
   virtual bool initInput();
+  virtual bool supportsKeyUpDown() { return false; }
```
Comment (tru) [inline src/input/InputLIRC.h]: All virtual here should be changed to `override` like everywhere else.

**Comment by plexappbuilder** [comment]
:balloon: - Good work!

**Pull request merged**